docs/build/provision/_index.md

```diff
@@ -0,0 +1,251 @@
+---
+title: "Provision Machines Using the Viam Agent"
+linkTitle: "Provision Machines"
+weight: 70
+type: "docs"
+description: "Flexibly provision new machines using the Viam Agent"
+tags: ["fleet management", "viam-server", "viam-agent"]
+# SME: James Otting
+---
+
+{{<gif webm_src="/platform/provisioning-demo.webm" mp4_src="/platform/provisioning-demo.mp4" alt="Using the Viam mobile app to provision a new machine with the Viam Agent" class="alignright" max-width="400px">}}
+
+You can use the _Viam Agent_ to provision a machine as it first comes online with a pre-defined configuration.
+This is useful when deploying a fleet of machines directly from the factory to a customer, or when bundling proprietary software on your Viam machine.
+You can install the Viam Agent on your machines as part of your build process, and then have the Viam Agent perform the rest of the first-time setup for your machine once deployed to a customer, or to the field.
+
+The Viam Agent:
+
+- Automatically connects to a pre-configured WiFi signal, or creates its own wireless hotspot if no pre-configured WiFi signal is detected.
+- Installs `viam-server` as a static binary, removing the need to perform any library linking or dependency installation during first-time setup.
+  You can also use a custom build of `viam-server`, if needed.
+- Provides automatic updates for `viam-server`, the agent itself, and any configured subsystems (such as the Agent Provisioning subsystem).
+- Allows control of deployed software versions through the Viam app.
+
+Consider a company that sells machines that monitor weather conditions on maritime craft and provide navigation advice based on those readings.
+Such a machine might use Viam to interface between a [sensor component](/components/sensor/) that takes weather measurements, and the [data management service](/data/) to regularly upload a stream of readings, for example.
+However, to then parse the readings and provide tailored guidance to a ship's captain, the company has written their own proprietary application which includes live analytics and speech generation for conveying advice to the captain.
+
+Using the Viam Agent, this company could ship their machines directly to customers and have each machine provision `viam-server` as it comes online for each user, eliminating factory setup time and allowing for tailored configurations per customer as needed.
+
+The Viam Agent is [open source](https://github.com/viamrobotics/agent), and is available for Linux machines running the `x86_64` and `arm64` architectures.
```

Comment:
FYI the Viam Agent is currently private, maybe we can hold on this line until we make it public.